src/components/Recommendation.module.css

```diff
@@ -3,3 +3,30 @@
   vertical-align: middle;
   margin: 0 10px;
 }
+
+.button:global(.ui.button) {
+  flex: auto !important;
+  color: var(--primary-color) !important;
+  background: #fff;
+}
+
+.button:global(.ui.active.button) {
+  background: #f2f4f2;
+}
+
+.button:global(.ui.active.button:hover) {
+  background: #fff;
+}
+
+.button:global(.ui.button:hover) {
+  background: #fff;
```

Comment:
```suggestion
  background: #f7f7f7;
```

Reply:
This brings a visible change in the color, does not seems to be constant with other buttons.